assets/search/header.ejs

```diff
@@ -5,13 +5,15 @@
       ng-submit="search.submit()">
   <input type="submit" style="position: absolute; left: -9999px"/>
 
-  <div class="form-group" ng-if="!onSearchPage && user">
+  <div class="form-group" ng-if="user">
```

Comment:
Right, can we keep onSearchPage here, because it feels incredibly unnecessary (and messy) to have it view on the search page.